src/solvers/strings/string_constraint_generator_format.cpp

```diff
@@ -299,7 +299,8 @@ add_axioms_for_format_specifier(
     return {res, std::move(return_code.second)};
   case format_specifiert::STRING:
   {
-    auto string_expr = get_string_expr(array_pool, get_arg("string_expr"));
+    const exprt arg_string = get_arg("string_expr");
```

Comment:
:pick: typo in commit message ("functtion"). Also there's a missing "of" I think.